frontend/src/pages/SubProjects/actions.js

```diff
@@ -101,9 +103,19 @@ export function setProjectHistoryOffset(offset) {
   };
 }
 
-export function fetchProjectHistory(projectId, offset, limit, showLoading = false) {
+export function fetchFirstProjectHistory(projectId, offset, limit, showLoading = false) {
```

Comment:
Please find a better name e.g. `fetchMostRecentProjectHistoryItems`.And why do I need an `offset` here? For the user in his domain it is only interesting to choose the amount of history items, and the ID of the project.